snforge_std/Scarb.toml

```diff
@@ -3,4 +3,4 @@ name = "snforge_std"
 version = "0.1.0"
 
 [dependencies]
-starknet = "2.3.1"
+starknet = "2.4.0"
```

Comment:
It is fine but please remember to not bump it in later bumps (it should stay set to the lowest compatible scarb version, otherwise the test collector thing is useless)